airbyte-integrations/connectors/source-pipedrive/source_pipedrive/streams.py

```diff
@@ -110,6 +110,12 @@ class Deals(PipedriveStream):
     """
 
 
+class DealFields(PipedriveStream):
+    """https://developers.pipedrive.com/docs/api/v1/DealFields#getDealFields"""
+
+    primary_key = None
```

Comment:
isn't the PK `id`?

Reply:
@sherifnada It should be. The only reason I have made them None is because the already existing ActivityFields stream had it as such and I though that maybe there is some intent behind it.

Reply:
According to ux best practices it's best if we make the decision for the user (if we know what's correct) so I'd suggest we set this in the code if we know what the right PK is